src/__tests__/index.js

```diff
@@ -1,10 +0,0 @@
-import { equal } from 'assert';
-import arrayTest from '../utils/index';
-
-describe('Array', () => {
```

Comment:
Why was this file deleted ?

Reply:
@OlawaleJoseph  you haven't reponded to this yet

Reply:
It was a dummy test created for coveralls purpose. We don't need it anymore. The reason for the deletion